src/qilib/utils/serialization.py

```diff
@@ -79,21 +79,21 @@ def _encode_bytes(data: Any) -> Dict[str, Any]:
 
 
 def _decode_bytes(data: Dict[str, Any]) -> bytes:
-    return data[JsonSerializeKey.CONTENT].encode('utf-8')
+    return bytes(data[JsonSerializeKey.CONTENT].encode('utf-8'))
```

Comment:
are we sure that we just want to/safely can cast the output to bytes?

Reply:
The result type of this method says we can.
Further: in builtins we see the signature:

    def encode(self, encoding='utf-8', errors='strict'):
        """
        S.encode(encoding='utf-8', errors='strict') -> **bytes**